### PR TITLE
[add]code3 swap-out/hibernation

### PR DESCRIPTION
**Pull request by MonkeyTedBaker**

### CODE REVIEW

File: monocle/worker.py [hunk 1]
```diff
@@ -769,9 +769,10 @@ def travel_speed(self, point):
         except ex.ServerBusyOrOfflineException as e:
             self.log.warning('{} Giving up.', e)
         except ex.BadRPCException:
-            self.error_code = 'BAD REQUEST'
             self.log.warning('Removing {} until the next run due to code 3 response.', self.username)
```
Comment (captainmedic) [inline monocle/worker.py]: May want to change wording now that we have decided on hibernating

Comment (MonkeyTedBaker) [reply monocle/worker.py]: Removing -> Hibernating?

Comment (captainmedic) [reply monocle/worker.py]: `Removing {} until the next run due to code 3 response.`
`Hibernating {} due to code 3 response.`

**Comment by MonkeyTedBaker** [comment]
Did some changes lol.

**Pull request merged**